x/mm2/types/utils.go

```diff
@@ -0,0 +1,30 @@
+package types
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+// ReadMarketConfigFromFile reads a market map configuration from a file at the given path.
+func ReadMarketConfigFromFile(path string) (MarketMap, error) {
```

Comment:
Can we rename this ?

`ReadMarketMapFromFile`

and test?

can just create issue to test at later time